aws-geo-location/src/main/java/com/amplifyframework/geo/location/AWSLocationGeoPlugin.kt

```diff
@@ -387,10 +385,10 @@ class AWSLocationGeoPlugin(
         GeoDeviceType.UNCHECKED -> id
         GeoDeviceType.USER_AND_DEVICE ->
             (credentialsProvider as CognitoCredentialsProvider).getIdentityId() + " - " +
-                sharedPreferences.getId()
-        GeoDeviceType.DEVICE -> sharedPreferences.getId()
+                locationTracker.database.sharedPreferences.getId()
```

Comment:
Hm using the shared preferences from the database class seems a little off. Perhaps it should be a shared preferences instance for the Geo category that is passed in to the database instead.